language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,944 @@
+{
```

Comment:
<p>Because this is a complex query, I'd actually suggest breaking this up into two or more cells:</p><p> </p><ol><li>The raw data, before passing it to the LLM.</li><li>The prompt template</li><li>Combining the raw data + prompt template + LLM call, using something like an f-string or &lt;string&gt;.format syntax</li></ol><p> </p><p>By doing so, you can differentiate between the before/after of the LLM call, and you also show how a user can carefully craft the prompt as a separate step before using it with the LLM </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>thanks for this - updated by Alicia</p>